maxim/dataset/dataset.py

```
@@ -109,3 +115,102 @@ def validate_data_structure(
+            List of dataset entries to add. Can be DatasetEntry objects or dictionaries.
+
+    Returns:
+        dict[str, Any]: Response data from the API
+
+    Raises:
+        TypeError: If entry type is not DatasetEntry or dict
+        Exception: If API call fails
+    """
+    total_rows = api.get_dataset_total_rows(dataset_id)
+
+    converted_entries: List[DatasetEntry] = []
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)